File: app/src/main/java/libertypassage/com/app/models/TrackUserLocationDetails.java

package libertypassage.com.app.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TrackUserLocationDetails {

    @SerializedName("user_details")
    @Expose
    private UserDetailsTrackUser userDetails;

    public UserDetailsTrackUser getUserDetails() {
        return userDetails;
    }

    public void setUserDetails(UserDetailsTrackUser userDetails) {
        this.userDetails = userDetails;
    }

}

File: app/src/main/java/libertypassage/com/corporate/model/TrackUserLocationDetails.kt
package libertypassage.com.corporate.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TrackUserLocationDetails {
    @SerializedName("timer_countdown")
    @Expose
    var timerCountdown: String? = null
    @SerializedName("user_details")
    @Expose
    var userDetails: UserDetailsTrackUser? = null
}